tracing/src/lib.rs

```diff
@@ -800,6 +800,7 @@
 //!    applications.
 //!  - [`tracing-elastic-apm`] provides a layer for reporting traces to [Elastic APM].
 //!  - [`tracing-etw`] provides a layer for emitting Windows [ETW] events.
+//!  - [`tracing-fluent-assertions`] provides a fluent assertions-style testing framework for validating the behavior of `tracing` spans.
```

Comment:
again, can we wrap this to match? sorry, i know this is silly.
```suggestion
//!  - [`tracing-fluent-assertions`] provides a fluent assertions-style testing 
//!    framework for validating the behavior of `tracing` spans.
```